runtime/doc/filetype.txt

```diff
@@ -544,6 +544,8 @@ CTRL-T                    Jump back to the location that the manpage was
                           opened from.
 q                         :quit if invoked as $MANPAGER, otherwise :close.
 
+To see the table of contents, type <M-]>.
```

Comment:
Any suggestions on how to display this.  The other keys are displayed as `CTRL-]` opposed to `<C-]>`. But,`META-]` and `ALT-]` doesn't look right and doesn't highlight.